cgo_flags_jemalloc.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: Marc Berhault (marc@cockroachlabs.com)
+
+// +build jemalloc
+
+package rocksdb
+
+// #cgo darwin CPPFLAGS: -I../c-jemalloc/darwin_includes/internal/include
+// #cgo linux CPPFLAGS: -I../c-jemalloc/linux_includes/internal/include
+// #cgo darwin LDFLAGS: -Wl,-undefined -Wl,dynamic_lookup
+// #cgo linux LDFLAGS: -Wl,-unresolved-symbols=ignore-all
+//
+// #include <jemalloc/jemalloc.h>
```

Comment:
Lgtm
On Apr 24, 2016 06:06, "marc" <notifications@github.com> wrote:

> In cgo_flags_jemalloc.go
> <https://github.com/cockroachdb/c-rocksdb/pull/19#discussion_r60842139>:
>
> > +// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
> > +// implied. See the License for the specific language governing
> > +// permissions and limitations under the License.
> > +//
> > +// Author: Marc Berhault (marc@cockroachlabs.com)
> > +
> > +// +build jemalloc
> > +
> > +package rocksdb
> > +
> > +// #cgo darwin CPPFLAGS: -I../c-jemalloc/darwin_includes/internal/include
> > +// #cgo linux CPPFLAGS: -I../c-jemalloc/linux_includes/internal/include
> > +// #cgo darwin LDFLAGS: -Wl,-undefined -Wl,dynamic_lookup
> > +// #cgo linux LDFLAGS: -Wl,-unresolved-symbols=ignore-all
> > +//
> > +// #include <jemalloc/jemalloc.h>
>
> Done.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/c-rocksdb/pull/19/files/a4b561f842e3eaa9eb283d5198d67ea87592fae2#r60842139>
>